app/views/users/index.haml

```diff
@@ -27,7 +27,7 @@
         %td= yes_no_image user.supervisor?
         %td= yes_no_image user.staff?
         %td
-          - if user.driver? && user.incident_reports.present?
+          - if user.driver?
```

Comment:
Correct, significant speed advantage not grabbing the incident reports, and it's perfectly valid to show an empty index.